crud/common.go

```diff
@@ -67,9 +69,9 @@ func newCall(method string) *tarantool.CallRequest {
 	return tarantool.NewCall17Request(method)
 }
 
-// Code returns IPROTO code for CRUD request.
-func (req baseRequest) Code() int32 {
-	return req.impl.Code()
+// Type returns IPROTO type for CRUD request.
+func (req baseRequest) Type() iproto.Type {
```

Comment:
Is this API public? Shouldn't we write down this change in changelog and migration guide as well?

Reply:
Added notes into changelog and readme.